Source/Core/CircleGeometry.js

```diff
@@ -73,7 +70,9 @@ define([
             vertexFormat : options.vertexFormat,
             stRotation : options.stRotation
         };
-        this._ellipseGeometry = new EllipseGeometry(ellipseGeometryOptions);
```

Comment:
To keep track of what properties are part of the object, we like to put `this._whatever = undefined;` in the constructor for properties that don't have a value.
It doesn't matter for the code, but it's just a convention we follow.